mariadb_kernel/maria_magics/load.py

```diff
@@ -0,0 +1,70 @@
+"""This class implements the %load magic command"""
+
+help_text = """
+The %load magic command has the following syntax:
+    > %load csv_file_path table_name [skip_row_num]
+The %load magic command can load CSV file for updating specific table data.
+
+This command does not create a table if the one specified as argument doesn't exist,
+the user needs to create the destination table with the proper schema to match the data in the CSV file.
+
+CSV file first line may be header, can set [skip row num] to 1 for skipping header.
+
+Any argument can be enclosed by ' ' or " ", handling cases that argument contains spaces.
+"""
+
+from mariadb_kernel.maria_magics.line_magic import LineMagic
+import shlex
+
+
+class Load(LineMagic):
+    def __init__(self, args):
+        self.args_list = shlex.split(args)
+
+    def name(self):
+        return "%load"
+
+    def help(self):
+        return help_text
+
+    def execute(self, kernel, data):
+        self.skip_row_num = 0
+
+        if len(self.args_list) < 2:
+            kernel._send_message(
+                "stderr",
+                "There was an error while parsing the arguments.\n"
+                + "Please check %lsmagic on how to use the magic command",
+            )
+            return
+        else:
+            self.csv_file_path = self.args_list[0]
+            self.table_name = self.args_list[1]
+            if len(self.args_list) > 2:
+                self.skip_row_num = int(self.args_list[3])
+
+        try:
+            open(self.csv_file_path).close()
+        except FileNotFoundError:
+            err = "CSV file not found"
```

Comment:
What happens if the file exists but the user doesn't have read permissions? Or if the file existed when the check ran, but was deleted later ([time of check to time of use](https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use))?

Best practice is to try and use the file, and show the error if that fails.
```suggestion
        try:
            open(self.csv_file_path).close()
        except IOError as e:
            err = f"Could not open CSV file at '{self.csv_file_path}': {e}"
```

But even this is not perfect, because what happens if the MariaDB process has access to a file, but Jupyter doesn't? I think the simplest option is to not check for the file, and let MariaDB handle the error messages.

Reply:
Good catch! @a97410985 we need to remove this try-except here. Because we use LOCAL in LOAD DATA, the MariaDB client will try to load the file itself (client is living on the same host as the jupyter kernel) so the mariadb client will do this error checking for us.